source/rfc/rfc-4.rst

```diff
@@ -8,7 +8,7 @@ RFC 4: GeoMoose Commit Management
 :Contact: bob DOT basques at ci DOT stpaul DOT mn DOT us
 :Contact: klassen DOT js at gmail DOT com
 :Status: Adopted
-:Last Updated: 2013-07-25
+:Last Updated: 2021-02-05
```

Comment:
I would probably leave the date as 2013-07-25 and link to this housekeeping commit.  Or possibly list both dates.  Oh, I see you more or less already address this at the bottom.

Reply:
Agree with Eli, as we are only scrubbing toxic language, I believe we can keep the date the same.